.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,88 @@
+name: Bug report
+description: Create a bug report
+labels: [ bug ]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of the bug, including steps to reproduce it and the normally expected behavior.
+        You can also attach screenshots or screen recordings to help explain your issue.
+      placeholder: |
+        1. Go to …
+        2. Click on …
+        3. Scroll down to …
+        4. See error / the app crashes
+
+        Instead, I expect …
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: |
+        Please paste your client logs (logcat) here, *NOT* server logs (in most cases).
+        Learn how to capture those logcats [here](https://wiki.lineageos.org/logcat.html).
+      placeholder: Paste logs…
+      render: shell
+  - type: input
+    id: app-version
+    attributes:
+      label: Applicaiton version
+      description: The version of the installed Jellyfin Android app.
+      placeholder: 2.3.0
+    validations:
+      required: true
+  - type: dropdown
+    id: installation-source
+    attributes:
+      label: Where did you install the app from?
+      description: Choose the appropriate app store or installtion method.
```

Comment:
```suggestion
      description: Choose the appropriate app store or installation method.
```

Reply:
The amount of typos is way too high. Not sure why Android Studio didn't detect them.

Reply:
Maybe you disabled the spellchecker? 🤔

Reply:
I don't think so? But maybe it didn't apply to that yaml file.